### ISSUE TITLE
Warmup learning rate scheduling

### ISSUE THREAD

**Issue by dssrgu** [kibok90/imix#1]
Hello, thanks for a great work!

On your code https://github.com/kibok90/imix/blob/6e149417f437ac93e7d710caf4cdf83620468611/main_pretext.py#L221-L224, shouldn't the `args.warmup_to` variable be set to

args.lr * (1 + math.cos(math.pi * args.warm_epochs / args.epochs)) / 2 

as this will make the warmup lr match the cos lr at the last warmup epoch?

**Comment by kibok90** [comment]
Yeah `eta_min` is supposed to be also in `adjust_learning_rate` but missing, so there is a non-continuous lr transition around the end of warmup. But the difference should be almost negligible anyway, as the actual value of `args.warmup_to` is relatively large.
